config.js

```diff
@@ -5,7 +5,7 @@ var config = {
   // Show help text for images
   showHelp: true,
   // Files to get data from
-  versions: {
+  releases: {
```

Comment:
How do you know it will be releases and not versions?

Reply:
yeah, I think version is better. I will change it back.